app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,83 @@
+require 'pry'
+class TweetsController < ApplicationController
+  
+  get '/tweets' do
+    if logged_in?
+      @tweets = Tweet.all
+      erb :'tweets/tweets'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets' do
+    #binding.pry
+
+    if params[:content].empty?
+      redirect '/tweets/new'
+    else
+      #binding.pry
+      @user = User.find(session[:user_id])
+      @tweet = Tweet.create(:content => params[:content], :user_id => @user.id)
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.